exonum-java-binding/testkit/README.md

```diff
@@ -0,0 +1,16 @@
+# TestKit
+
+Exonum Java Binding provides a powerful testing toolkit - TestKit.
+TestKit allows testing transaction execution in the synchronous environment by offering simple
+network emulation (that is, without consensus algorithm and network operation involved).
+
+For more details and a guide on how to use TestKit in your tests, see
+[documentation][exonum-testkit].
```

Comment:
Mb it'll be a good idea to point users to cryptocurrency demo tests that use `TestKit` as a live examples?

Reply:
Maybe, to the end of the documentation section on testkit?